tests/foreman/api/test_errata.py

```diff
@@ -513,8 +476,62 @@ def test_positive_filter_by_envs(module_org):
     assert len(errata_library) > len(errata_env)
 
 
+@pytest.fixture(scope='module')
+def setup_content_rhel6():
```

Comment:
A small docstring should help !

Also, you can move this to global fixtures !

Reply:
Doc string added.
Followup PR for making it global